templates/blockonomics_crypto_options.php

```diff
@@ -4,30 +4,28 @@
 $order_id = $_REQUEST['select_crypto'];
 $order_url = $blockonomics->get_parameterized_wc_url(array('show_order'=>$order_id))
 ?>
-<div class="bnomics-order-container">
+<div class="woocommerce bnomics-order-container">
   <div class="bnomics-select-container">
-    <table width="100%">
-      <tr>
-        <?php
-        foreach ($cryptos as $code => $crypto) {
-          $order_url = add_query_arg('crypto', $code, $order_url);
-        ?>
-          <td class="bnomics-select-options">
-            <a href="<?php echo $order_url;?>" style="color: inherit; text-decoration: inherit;">
-              <p>
-                <?=__('Pay With', 'blockonomics-bitcoin-payments')?>
-              </p>
-              <span class="bnomics-icon-<?php echo $code;?> bnomics-rotate-<?php echo $code;?>"></span>
-              <p>
-                <?php echo $crypto['name'];?><br>
-                <b><?php echo $code;?></b>
-              </p>
-            </a>
-          </td>
-        <?php 
-        }
-        ?>
-      </tr>
-    </table>
+    <tr>
+      <?php
+      foreach ($cryptos as $code => $crypto) {
+        $order_url = add_query_arg('crypto', $code, $order_url);
+      ?>
+        <a href="<?php echo $order_url;?>">
+          <input type="hidden" name="show_order" value="<?php echo $order_id;?>" />
+          <input type="hidden" name="crypto" value="<?php echo $code;?>" />
```

Comment:
As we are now using an anchor and not a form the hidden inputs are not required. $order_url contains the show_order and crypto params.

Reply:
Removed the hidden inputs